Functions/Quantification/xASL_quant_FSL.m

```diff
@@ -95,8 +95,13 @@
     end
 
     %% 4. Create option_file that contains options which are passed to the FSL command
-	% FSLOptions is a character array containing CLI args for the BASIL/FABBER command
-	FSLOptions = xASL_sub_FSLOptions(pathFSLOptions, x, bUseFabber, PWI, pathFSLInput, pathFSLOutput);
+	% To be removed later when we are properly checking for merged multi-TE images
+	if x.modules.asl.bMergingSessions == 1
+		bUseFabber = 1;
```

Comment:
Please can you separate session merging and the use of FABBER. E.g., Mervin will use session merging for all Singapore studies, where most are without DEBBIE.